actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -55,6 +61,10 @@ def with_broadcast_connection(&block) # :nodoc:
         end
       end
 
+      def compress_payloads?
```

Comment:
This wont memoize correctly if it returns false. Do you even need to memoize here?

Reply:
You're right, fixed